CHANGELOG.md

```diff
@@ -1,4 +1,8 @@
-### 0.3.5 (Next)
+### 0.3.6 (Next)
+
+* Your contribution here.
+
+### 0.3.5 (04/29/2019)
```

Comment:
Don't add a version, just add your changelog line.

Reply:
sorry about that.